cypress/integration/layout-spec.js

```diff
@@ -0,0 +1,48 @@
+import { Preview } from '../page-objects/preview-page'
+import { goOffline, goOnline } from '../page-objects/helpers/network'
+
+const preview = new Preview()
+
+describe( 'Check the layout', () => {
+
+	beforeEach( 'Open the Test Page', () => {
+		cy.navigateToTestPage()
+	} )
+
+	it( 'Check the Wikipedia Preview in Standard with Image Layout', () => {
+		preview.getPreviewSpan().eq( 0 ).click()
+
+		preview.checkPreviewStandard()
+
+		preview.getHeaderClosebtn().click()
+	} )
+
+	it.skip( 'Check the Wikipedia Preview in Offline Layout', () => {
```

Comment:
Why are the next 3 tests skipped?

Reply:
WIP for the next phases of work since the offline mode doesn't work as expected on cypress